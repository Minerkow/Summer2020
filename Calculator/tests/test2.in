a = ?;
yjnt  = a * 805 + ((688 +((265 -((618 *(761 -630 )+942 )+38 +899 )))));
b = ?;
c = ?;
pwlb  = b + c -((588 +(170 *((131 /((274 +511 )))))/956 )+210 )*7;
rggm  = 1 /((808 /(37 +(717 /(423 )+844 )*602 )+390 *474 -(770 -573 )))*587 -565 ;
print yjnt;
print pwlb;
print rggm;