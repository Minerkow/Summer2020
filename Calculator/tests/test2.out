-79469
-5578
-565
